# CWWBA0142E An exception occurred while handling a human task that is associated with object ''0'' in process ''1''.

## Explanation

## User response